src/dataprotection/azext_dataprotection/generated/custom.py

```diff
@@ -1,301 +1,319 @@
+                       vault_name=vault_name,
+                       resource_group_name=resource_group_name,
+                       parameters=parameters)
+
+
+def dataprotection_backup_instance_validate_for_restore(client,
+                                                        vault_name,
+                                                        resource_group_name,
+                                                        backup_instance_name,
+                                                        restore_request_object,
+                                                        no_wait=False):
+    parameters = {}
+    parameters['restore_request_object'] = restore_request_object
+    return sdk_no_wait(no_wait,
+                       client.begin_validate_for_restore,
+                       vault_name=vault_name,
+                       resource_group_name=resource_group_name,
+                       backup_instance_name=backup_instance_name,
+                       parameters=parameters)
+
+
+def dataprotection_recovery_point_list(client,
+                                       vault_name,
+                                       resource_group_name,
+                                       backup_instance_name,
+                                       filter_=None,
+                                       skip_token=None):
+    return client.list(vault_name=vault_name,
+                       resource_group_name=resource_group_name,
+                       backup_instance_name=backup_instance_name,
+                       filter=filter_,
+                       skip_token=skip_token)
+
+
+def dataprotection_recovery_point_show(client,
+                                       vault_name,
+                                       resource_group_name,
+                                       backup_instance_name,
+                                       recovery_point_id):
+    return client.get(vault_name=vault_name,
+                      resource_group_name=resource_group_name,
+                      backup_instance_name=backup_instance_name,
+                      recovery_point_id=recovery_point_id)
+
+
+def dataprotection_job_list(client,
+                            resource_group_name,
+                            vault_name):
+    return client.list(resource_group_name=resource_group_name,
+                       vault_name=vault_name)
+
+
+def dataprotection_job_show(client,
+                            resource_group_name,
+                            vault_name,
+                            job_id):
+    return client.get(resource_group_name=resource_group_name,
+                      vault_name=vault_name,
+                      job_id=job_id)
+
+
+def dataprotection_restorable_time_range_find(client,
+                                              vault_name,
+                                              resource_group_name,
+                                              backup_instance_name,

```

Comment:
Why the parameter changed from backup_instance to backup_instance_name

Reply:
Earlier the parameter name was backup_instances. It was misleading to users for two reasons :

1. It had plural naming(has 's' in the end). But the parameter was only expecting a single input.
2. backup_instances was giving an impression that the parameter is expecting an object/JSON file but instead it was expecting the name of the backup instance.

Also, the naming was inconsistent with respect to other commands which were also expecting backup_instance_name as parameter input. So this correction was made in the swagger. 

I think it would be better if we allow this change now as the module is in preview and not yet GA.  Otherwise, we can use directives to change it back to backup_instances but that in my opinion will cause confusion and inconsistency. Please share your thoughts.

Reply:
I see. Please update the version of this extension in manual/versions.py and add history notes in history.rst

Reply:
Did the changes. Please have a look.